web_external/models/ItemModel.js

```diff
@@ -0,0 +1,15 @@
+isic.models.ItemModel = girder.models.ItemModel.extend({
+    /**
+     * Get the files within the item.
+     */
+    getFiles: function () {
```

Comment:
Since `Item` is not typically something we deal with directly in ISIC, I think it'd be better to patch this method onto the `girder.models.ItemModel` instead (or maybe the prototype, I forget how Backbone inheritance works). Eventually, we should not need this code anyway, once the uploader widget does what we want it to.